src/components/ArmyBuildPane.jsx

```diff
@@ -13,12 +16,34 @@ const Container = styled.div`
     margin: 30px
 `;
 
-const ArmyBuildPane = () => (
-  <Container>
-    <UsageBar />
-    <SelectionPane />
-    <ArmyList />
-  </Container>
-);
+const ArmyBuildPane = () => {
+  const [unitList, setUnitList] = useState({});
+  const [listIndex, setListIndex] = useState(1);
```

Comment:
What is the semantic difference between these too? (non-code question)

Reply:
Basically unitList is a collection of what the user has added. It's an object instead of an array so I can reference keys for deletion and modification. listIndex/setListIndex is really just a helper to ensure I have a unique key for each selection. Definitely interested in other approaches to generating a key. 

Originally considered making it a hash based on selection, but most units you can select more than once so had to throw that out.

Reply:
> listIndex/setListIndex is really just a helper to ensure I have a unique key for each selection
What is a 'selection'? Sounds like a 'pending confirm' state, almost like it's in your cart but hasn't been purchased yet. Is that right or not quite? If so, it might be better to have a separate "cart" state type of thing. If not, then sorry that I don't fully understand it yet and I'm sure you've got a good thing going 😅

Reply:
Or is the list index referring to commander vs operative etc?